### DISCUSSION TITLE
LangChain Error

### DISCUSSION THREAD

**Discussion by thc-droid** [logancyang/obsidian-copilot#546] [Q&A]
I entered the API key, installed the 3.5 version, and saved the changes. It gives such an error. There's nothing wrong with the Internet.
![Снимок экрана 2024-04-17 234715](https://github.com/logancyang/obsidian-copilot/assets/146643703/4abb5529-2236-4583-b490-9247c3c58691)

**Comment by iukea1** [comment]
Sounds like https://github.com/logancyang/obsidian-copilot/issues/382

**Comment by runningZ1** [comment]
![微信图片_20240428162829](https://github.com/logancyang/obsidian-copilot/assets/127315512/bd139484-6cbf-47a7-881a-92ec85211a1a)

In the mode of asking questions about the entire knowledge base, I ran into the same problem

**Comment by jimbo108108** [comment]
I'm getting the same problem. Has the problem been resolved? I'm running llama3 on Mac Sonoma. The specific error message I'm getting is: Langchain error: TypeError: Failed to fetch.
